lib/rules/no-restricted-browser-globals-during-ssr.js

```diff
@@ -0,0 +1,844 @@
+    'SVGScriptElement',
+    'SVGSetElement',
+    'SVGStopElement',
+    'SVGStringList',
+    'SVGStyleElement',
+    'SVGSwitchElement',
+    'SVGSymbolElement',
+    'SVGTSpanElement',
+    'SVGTextContentElement',
+    'SVGTextElement',
+    'SVGTextPathElement',
+    'SVGTextPositioningElement',
+    'SVGTitleElement',
+    'SVGTransform',
+    'SVGTransformList',
+    'SVGUnitTypes',
+    'SVGUseElement',
+    'SVGViewElement',
+    'Scheduler',
+    'Scheduling',
+    'Screen',
+    'ScreenOrientation',
+    'ScriptProcessorNode',
+    'SecurityPolicyViolationEvent',
+    'Selection',
+    'ShadowRoot',
+    'SharedWorker',
+    'SourceBuffer',
+    'SourceBufferList',
+    'SpeechSynthesisErrorEvent',
+    'SpeechSynthesisEvent',
+    'SpeechSynthesisUtterance',
+    'StaticRange',
+    'StereoPannerNode',
+    'Storage',
+    'StorageEvent',
+    'StylePropertyMap',
+    'StylePropertyMapReadOnly',
+    'StyleSheet',
+    'StyleSheetList',
+    'SubmitEvent',
+    'SyncManager',
+    'TaskAttributionTiming',
+    'TaskController',
+    'TaskPriorityChangeEvent',
+    'TaskSignal',
+    'Text',
+    'TextDecoderStream',
+    'TextEncoderStream',
+    'TextEvent',
+    'TextMetrics',
+    'TextTrack',
+    'TextTrackCue',
+    'TextTrackCueList',
+    'TextTrackList',
+    'TimeRanges',
+    'Touch',
+    'TouchEvent',
+    'TouchList',
+    'TrackEvent',
+    'TransformStream',
+    'TransformStreamDefaultController',
+    'TransitionEvent',
+    'TreeWalker',
+    'TrustedHTML',
+    'TrustedScript',
+    'TrustedScriptURL',
+    'TrustedTypePolicy',
+    'TrustedTypePolicyFactory',
+    'UIEvent',
+    'URLPattern',
+    'UserActivation',
+    'VTTCue',
+    'ValidityState',
+    'VideoColorSpace',
+    'VideoFrame',
+    'VideoPlaybackQuality',
+    'VirtualKeyboardGeometryChangeEvent',
+    'VisualViewport',
+    'WaveShaperNode',
+    'WebGL2RenderingContext',
+    'WebGLActiveInfo',
+    'WebGLBuffer',
+    'WebGLContextEvent',
+    'WebGLFramebuffer',
+    'WebGLProgram',
+    'WebGLQuery',
+    'WebGLRenderbuffer',
+    'WebGLRenderingContext',
+    'WebGLSampler',
+    'WebGLShader',
+    'WebGLShaderPrecisionFormat',
+    'WebGLSync',
+    'WebGLTexture',
+    'WebGLTransformFeedback',
+    'WebGLUniformLocation',
+    'WebGLVertexArrayObject',
+    'WebKitCSSMatrix',
+    'WebKitMutationObserver',
+    'WebSocket',
+    'WheelEvent',
+    'Window',
+    'WindowControlsOverlay',
+    'WindowControlsOverlayGeometryChangeEvent',
+    'Worker',
+    'WritableStream',
+    'WritableStreamDefaultController',
+    'WritableStreamDefaultWriter',
+    'XMLDocument',
+    'XMLHttpRequest',
+    'XMLHttpRequestEventTarget',
+    'XMLHttpRequestUpload',
+    'XMLSerializer',
+    'XPathEvaluator',
+    'XPathExpression',
+    'XPathResult',
+    'XSLTProcessor',
+    'alert',
+    'blur',
+    'cancelAnimationFrame',
+    'cancelIdleCallback',
+    'captureEvents',
+    'chrome',
+    'clientInformation',
+    // 'close',
```

Comment:
We chatted about this on Slack, but also leaving this here for other reviewers & for posterity:

In the future, we can use ESlint's scope API to determine whether a variable is locally scoped or a global. When that is in place, we can uncomment these global variables with common names.